apps/src/templates/instructions/codeReview/Comment.jsx

```diff
@@ -50,18 +62,28 @@ export default class Comment extends Component {
       >
         <div style={styles.commentHeaderContainer}>
           {this.renderName()}
-          <span
+          <div
             className="fa fa-ellipsis-h"
             style={styles.ellipsisMenu}
             onClick={() =>
-              this.setState({showEllipsisMenu: !this.state.showEllipsisMenu})
+              this.setState({
+                isShowingCommentOptions: !isShowingCommentOptions
+              })
             }
-          />
+          >
+            {isShowingCommentOptions && (
+              <CommentOptions
+                isResolved={isResolved}
+                onResolveClick={() => {
```

Comment:
This should be moved into a helper (like we have for `onDeleteClick`) -- I've fixed this in the immediate follow-up PR. [insert link when fixed]